docs/advanced/gpu.md

```diff
@@ -0,0 +1,296 @@
+---
+title: Edge Pods use GPU
+sidebar_position: 9
+---
+
+## Abstract
+
+With the development of edge AI, the demand for deploying GPU applications on edge nodes is gradually increasing. Currently, KubeEdge can manage GPU nodes through some configurations, 
+and allocate GPU resources to user edge applications through the k8s device-plugin component. If you need to use this feature, please refer to the steps below.
+
+## Getting Started
+
+### GPU running environment construction
+
+Using GPU on edge nodes requires building a GPU operating environment first, which mainly includes the following steps:
+
+1. Install GPU driver
+
+First you need to determine whether the edge node machine has GPU. You can use the `lspci | grep NVIDIA` command to check. Download the appropriate GPU driver according to the specific GPU model and complete the installation. 
```

Comment:
so it turned out that this is only for Nvidia GPUs? if that is so, probably we could change the title into `How to enable Nvidia GPUs on KubeEdge` or something else?